config/locales/client.en.yml

```diff
@@ -29,7 +29,7 @@ en:
         title: "Endorse user"
         subtitle: "What are %{username}'s strengths?"
       review:
-        approve: "Approve..."
```

Comment:
I figured we remove these ominous dots, especially since this is not a drop-down.

Reply:
Ah, the "It's fine… Approve, I guess…" button! 😄